zip.sh

```diff
@@ -0,0 +1,7 @@
+npm run compile
+npm run copy-package
+cd build
+npm install --production
+rm ./*.json
```

Comment:
Probably don't need to do this? (Might cause some issues later)

Reply:
+1- on Jenkins I'm always starting from scratch (removing workspace), so it's being handled on it's own.